Loenn/entities/confetticontrollers.lua

```diff
@@ -39,8 +39,7 @@ confetti.fieldInformation = {
         editable = true
     },
     colors = {
-        allowXNAColors = true,
-        fieldType = "color"
+        allowXNAColors = true

```

Comment:
Currently unable to check, but does `allowXNAColors`  prevent the user from adding a comma-separated list? If so, this should be removed for all `colors` options (fields that allow comma-separated lists).

Reply:
Tested and Lönn allows a string of colors or an XNA color

Reply:
Nice, that's good!